packages/vite/src/node/server/hmr.ts

```diff
@@ -20,6 +21,7 @@ export interface HmrOptions {
   path?: string
   timeout?: number
   overlay?: boolean
+  server?: Server
```

Comment:
Maybe this should be `http.Server` instead (which is then passed to `createWebScoketServer` as the first argument)? It feels weird to require the user to create a ws server.

Reply:
Oh nvm..